services/forest/src/service/SCIHF.js

```diff
@@ -0,0 +1,162 @@
+module.exports = (SCIHFPersistence, restAPI) => {
+  const SCIHF = {
+    /**
+     * Get values for the forest structural condition index crossed with human footprint
+     * and protected area categories for a given area
+     *
+     * @param {String | Number} areaType area type
+     * @param {String} areaId area id
+     *
+     * @returns {Object[]} Values of Forest Structural Condition Index crossed with human footprint
+     * and protected area categories
+     */
+    getSCIHF: async (areaType, areaId) => {
+      let data;
+      switch (areaType) {
+        case 'ea':
+          data = await SCIHFPersistence.findSCIHFInEA(areaId);
+          break;
+        case 'states':
+          data = await SCIHFPersistence.findSCIHFInState(areaId);
+          break;
+        case 'basinSubzone':
+          data = await SCIHFPersistence.findSCIHFInBasinSubzone(areaId);
+          break;
+        case 'pa': {
+          const binaryProtected = await restAPI.requestBinaryProtectedByCategory(areaId);
+          data = await SCIHFPersistence.findSCIHFInPA(binaryProtected.binary_protected);
+          break;
+        }
+        default:
+          data = null;
+          break;
+      }
+
+      if (!data) {
+        throw new Error('Data for SCIHF doesn\'t exists to the selected area id and area type');
+      }
+
+      try {
+        const binaryProtectedValues = [...new Set(data.map(b => (b.binary_protected)))].join(';');
+        const categoriesPA = await restAPI.requestCategoriesByBinaryProtected(
```

Comment:
Adding this try-catch is fine, but the error I was getting before would still pass this, because the rest module just sends the response it's getting in json format.

Would be good to check the status in the response and only resolve the promise with the body if it's 200

Reply:
Done